### PR TITLE
Enhancement: CtD benefits section

### LINKED ISSUE #15: [Enhancement] Compute-to-Data Image
## Motivation / Problem
grafik

- From an UX perspective it is not fully clear, that the image is extendable / resizeable
- The resized picture (bigger) needs better resolution

## Solution 
- add the `zoom-in` cursor on hover
- Use bigger resolution for CtD graphic

## Additional context
Please add the graphic below in a comment @MeikeMolitor

### PR DESCRIPTION
**Pull request by LucaMilanese90**
Fixes #15 

## Proposed Changes

  - add zoom-in/out cursor on hover for CtD image
  - higher resolution CtD image

### CODE REVIEW

File: src/components/organisms/HomeContent.tsx [hunk 1]
```diff
@@ -72,15 +73,18 @@ export default function HomeContent(): ReactElement {
     ),
     image: (
       <>
-        <div onClick={() => setIsOpen(true)}>
-          <Img fluid={ctdBenefitsImage} />
+        <div className={styles.imagePreview} onClick={() => setIsOpen(true)}>
+          <img src={ctdBenefitsImage} alt="Compute-to-Data and it's benefits" />
```
Comment (moritzkirstein) [inline src/components/organisms/HomeContent.tsx:77]: I just noticed: it actually needs to say : "Compute-to-Data and **its** benefits"

File: src/components/organisms/HomeContent.tsx [hunk 2]
```diff
@@ -72,15 +73,18 @@ export default function HomeContent(): ReactElement {
...
         </div>
         <Modal isOpen={modalIsOpen} onRequestClose={() => setIsOpen(false)}>
           <div className={styles.modal} onClick={() => setIsOpen(false)}>
             <button id={styles.close} onClick={() => setIsOpen(false)}>
               close
             </button>
-            <Img fluid={ctdBenefitsImage} />
+            <img
+              src={ctdBenefitsImage}
+              alt="Compute-to-Data and it's benefits"
```
Comment (moritzkirstein) [inline src/components/organisms/HomeContent.tsx:86]: see above

**Review by moritzkirstein — CHANGES_REQUESTED** [review]
Can you please also adjust the header of the image section:
"Compute-to-Data and **its** benefits"

**Review by moritzkirstein — COMMENTED** [review]
Thanks for the adjustment. Please change the title of the section, too 🙏

**Comment by LucaMilanese90** [comment]
> Thanks for the adjustment. Please change the title of the section, too 🙏

Sorry for that 😅

**Review by moritzkirstein — APPROVED** [review]

**Pull request merged**